test/automated-suite.js

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
This is basically the same script used in sw-toolbox now, right?

Probably time to spin up a separate repo of helpers that we can reuse. Won't block this PR on it, but I'm also just going to assume that it is identical and not review it again here.